documentation/modules/appendix/pages/best-practices.adoc

```diff
@@ -12,56 +12,854 @@ SPDX-License-Identifier: MPL-2.0
-  On the other hand, abbreviating "minimum" and "maximum" with "min" and "max", respectively, is
-  fine. The Aspect Model should be reasonably expressive. Do not add alleged premature optimizations
-  that belong in the Aspect implementation: if you think a Property is name expressive but too long
-  for the runtime payload, consider adding xref:ROOT:modeling-guidelines.adoc#payload-names[a payload
-  name] to the Property's usage instead of renaming the Property.
-
-* Do not use redundant naming parts. In particular, the name of an Aspect should not end with
-  "Aspect". It is however fine to distinguish between e.g. an Entity and a related Characteristic
-  by a suffix when necessary.
-
-* Do not use redundant prefixes, e.g. when authoring an Aspect Model for the domain _Foo_, do not
-  name your Properties _fooCount_, _fooId_ etc. Remember that the full identifier of a Property is
-  the versioned namespace concatenated with the element name, e.g.
-  `urn:samm:{example-ns}.myapplication:1.0.0#count`, so the context is already part of
-  `count`&#8203;'s identifier.
-
-* The attributes `preferredName` and `description` are meant to provide a
-  human-readable name and
-  description, respectively, for example for the generation of documentation from the Aspect Model.
-  If you copy and paste the element's name to the `preferredName`, make sure to adjust it:
-** Make the name upper case
-** If the name consists of more than one word (i.e., camelCase), turn it into two words, e.g. the name
-   `machineTemperature` should become the preferredName `Machine Temperature`.
-
-* If there is exactly one Property in an Aspect that has a
-xref:ROOT:characteristics.adoc#collection-characteristic[Collection],
-xref:ROOT:characteristics.adoc#list-characteristic[List],
-xref:ROOT:characteristics.adoc#set-characteristic[Set],
-xref:ROOT:characteristics.adoc#sorted-set-characteristic[Sorted Set] or
-xref:ROOT:characteristics.adoc#time-series-characteristic[Time Series] Characteristic (and no other
-Properties), the Property should be named `items`.
-
-* If there is exactly one Property in an Aspect that has a
-  xref:ROOT:characteristics.adoc#collection-characteristic[Collection],
-  xref:ROOT:characteristics.adoc#list-characteristic[List],
-  xref:ROOT:characteristics.adoc#set-characteristic[Set] or
-  xref:ROOT:characteristics.adoc#sorted-set-characteristic[Sorted Set] Characteristic (and no other
-  Properties), the Aspect's name should be in plural. Otherwise, if there is exactly one Property in
-  the Aspect that has a xref:ROOT:characteristics.adoc#time-series-characteristic[Time Series]
-  Characteristic, the Aspect's name should end with "History". Otherwise, the Aspect's name should
-  be in singular.
+The following best practices, while not mandatory, are highly recommended for optimal performance and to mitigate potential issues, especially when working with Aspect Models at scale.
+
+[[the-name-attribute]]
+== The `name` attribute
```

Comment:
There is no "name" attribute any longer,at least it is different from an attribute "preferredName" etc.